en/02_Developer_Guides/03_Forms/01_Validation.md

```diff
@@ -394,6 +394,39 @@ class MyPage extends Page
 > You can also update the `CompositeValidator` by creating an `Extension` and implementing the
 > `updateCMSCompositeValidator()` method.
 
+### `RequiredFields` and whitespace
+
+By default, `RequiredFields` will consider a field with only whitespace as a valid value. You an change this behavior with the [`allow_whitespace_only`](api:SilverStripe\Forms\RequiredFields->allow_whitespace_only) global configuration, or on a per-instance basis using [`setAllowWhitespaceOnly()`](api:SilverStripe\Forms\RequiredFields::setAllowWhitespaceOnly()).
+
+```yml
+# global configuration
+SilverStripe\Forms\RequiredFields:
+  allow_whitespace_only: false
+```
+
+```php
+namespace App\PageType;
+
+use Page;
+use SilverStripe\Forms\CompositeValidator;
+use SilverStripe\Forms\RequiredFields;
+
+class MyPage extends Page
+{
+    // ...
+
```

Comment:
```suggestion
```
https://docs.silverstripe.org/en/5/contributing/documentation/#code-blocks
> Don't place a blank line after this comment, unless the next line is also a comment.

Reply:
Sorry, just saw that that doc reference is for something else.
This blank line is fine, feel free to leave it.
My brain is not all-in today lol